docs/workflows.rst

```diff
@@ -64,3 +67,59 @@ All the commands should be run on Pulp 3 machine.
             }
         ]
     }
+
+
+Reset migrated Pulp 3 data
+--------------------------
+
+.. note::
+    Nothing is getting reset or modified in any way in Pulp 2.
+
+.. note::
+    Migrated artifacts are untouched. If you are sure, you want to remove them, please run the
+    orphan cleanup task in Pulp 3.
+
+There are cases, when one needs to run a migration from scratch (by default, it's always
+incremental). E.g. some issue happened during the pulp2to3 migration which you are not able to
+recover from.
+
+All the commands should be run on Pulp 3 machine.
+
+
+1. Create a :doc:`Migration Plan <../migration_plan>` which you want to run a migration for.
```

Comment:
s/which you want to run a migration for/for the plugins or set of repositories you would like to migrate/